### PR TITLE
Tahoma Linux builds via Travis

### PR DESCRIPTION
**Pull request by manongjohn**
This PR sets up for building Tahoma for Linux via Travis if you wish to allow it.  It is also useful for linux users making local builds if they want to.

I purposefully modified to build with Qt 5.9 because I know there were issues with the OTX Linux version when I tried to build with 5.14 or later.

### CODE REVIEW

File: toonz/sources/xdg-data/io.github.Tahoma.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.Tahoma</id>
+  <launchable type="desktop-id">io.github.Tahoma.desktop</launchable>
+  <name>Tahoma</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-3-Clause</project_license>
+  <developer_name>DWANGO Co., Ltd.</developer_name>
```
Comment (turtletooth) [inline toonz/sources/xdg-data/io.github.Tahoma.appdata.xml:8]: Would you mind changing the developer?  Tahoma is fine.

File: toonz/sources/xdg-data/io.github.Tahoma.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    </p>
+    <p>
+      It is based on the software "Toonz", which was developed
+      by Digital Video S.p.A. in Italy, customized by Studio Ghibli,
+      and has been used for creating its works for many years.
+      Dwango launched the OpenToonz project in cooperation
+      with Digital Video and Studio Ghibli.
+    </p>
+  </description>
+  <url type="homepage">https://tahoma.github.io/e/</url>
```
Comment (turtletooth) [inline toonz/sources/xdg-data/io.github.Tahoma.appdata.xml:22]: https://tahoma2d.org

File: toonz/sources/xdg-data/io.github.Tahoma.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  <url type="bugtracker">https://github.com/turtletooth/tahoma/issues</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/flathub/io.github.Tahoma/master/screenshot1.png</image>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1" />
+  <releases>
+    <release version="1.4.0" date="2020-01-31"></release>
```
Comment (turtletooth) [inline toonz/sources/xdg-data/io.github.Tahoma.appdata.xml:31]: Release 1.0

File: toonz/sources/xdg-data/io.github.Tahoma.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-3-Clause</project_license>
+  <developer_name>DWANGO Co., Ltd.</developer_name>
+  <summary>2D animation</summary>
+  <description>
+    <p>
+      This is software for producing a 2D animation.
+    </p>
+    <p>
+      It is based on the software "Toonz", which was developed
```
Comment (turtletooth) [inline toonz/sources/xdg-data/io.github.Tahoma.appdata.xml:15]: Tahoma is based on OpenToonz which itself is based on Toonz, developed by Digital Video.

File: toonz/sources/xdg-data/io.github.Tahoma.appdata.xml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  <id>io.github.Tahoma</id>
+  <launchable type="desktop-id">io.github.Tahoma.desktop</launchable>
+  <name>Tahoma</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-3-Clause</project_license>
+  <developer_name>DWANGO Co., Ltd.</developer_name>
+  <summary>2D animation</summary>
+  <description>
+    <p>
+      This is software for producing a 2D animation.
```
Comment (turtletooth) [inline toonz/sources/xdg-data/io.github.Tahoma.appdata.xml:12]: This is software for producing 2D and stop motion animation.

**Comment by turtletooth** [comment]
Thanks for doing this.  Would this create an artifact of sorts?  Or does it automatically push to the releases?

**Comment by manongjohn** [comment]
I'll submit the changes shortly.

I don't have it configured to generate any artifact, though it can be set up to do it, however, unless something has changed, Travis can't create artifacts for individual PRs.  It can only create them for Releases after some configuration in `.travis.yml`.  Considering you are building releases manually due to the Canon support, it's probably not something you want.

My understanding is you aren't planning on doing a Linux version...yet, but I wanted to put this out there for future use.

Aside:

I noticed your PR checks have 2 line items for Travis builds...`Travis CI` (via travis-ci.**com**) and `continuous-integration/travis-ci/pr` (via travis-ci.**org**).  You are building 2 sets of Travis builds on 2 different servers.  You don't need both.  That could burn your GIT LFS bandwidth quickly.

Also, it's probably not really necessary to build both Linux gcc and clang versions.  I reduced my OTX Linux builds just to clang.  

Additionally, for Appveyor, I only build release versions.  Not sure the Debug version is worth building.

File: toonz/sources/xdg-data/io.github.Tahoma.appdata.xml (hunk 4 above)
Comment (manongjohn) [reply toonz/sources/xdg-data/io.github.Tahoma.appdata.xml:15]: Should I replace the entire text with this?

`Tahoma is based on OpenToonz which itself is based on "Toonz", developed by Digital Video S.p.A. in Italy and customized by Studio Ghibli.`

or just the line you gave me?

Comment (turtletooth) [reply toonz/sources/xdg-data/io.github.Tahoma.appdata.xml:15]: I’m fine with either. Thanks.

**Comment by turtletooth** [comment]
>  Considering you are building releases manually due to the Canon support, it's probably not something you want.
You are right.  And right now Canon doesn't make a SDK for linux.  I'm curious what DragonFrame is using on the linux side.

Thanks for the tips on travis and appveyor.  I'll make those changes.

Appreciate your work on this.

**Pull request merged**